testdata/p4_16_dpdk_errors_outputs/psa-dpdk-128bitCast_error.p4-error

```diff
@@ -0,0 +1,3 @@
+psa-dpdk-128bitCast_error.p4(55): [--Werror=overlimit] error: DPDK target supports up-to 64-bit immediate values, 80w0xffffbbbb12345342aaaa exceeds the limit
```

Comment:
I am a bit confused by this error message.  These changes are to enable support in DPDK back end for 128-bit immediate values, yes?

If so, then the error message seems misleading.  It seems best either to (a) support this assignment, or (b) give a more precise error message, whatever the restriction might actually be.  Is the restriction given these changes something like "DPDK target supports immediate values that are up to 64-bit, and also 128-bit, but no other sizes.  80w0xffffbbbb12345342aaaa is not a supported size" ?